packages/storybook/stories/segmented-control.mdx

```diff
@@ -0,0 +1,40 @@
+import { ToggleButtonGroup } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as ToggleButtonGroupStories  from './segmented-control.stories';
+
+<Meta of={ToggleButtonGroupStories} />
+
+# Segmented Control
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+A segmented control is a horizontal set of 2 to 5 exclusive segments, each functioning as a mutually exclusive button. Segmented controls provide closely related choices that affect an object, state, or view [2][3][4]. 
+<Canvas of={ToggleButtonGroupStories.Default} />
+
+## Usage
+### When to use
+- Use when the change has an immediate effect on the UI [2].
+- Use when the options function as mutually exclusive buttons [2].
+- Use if your options are short and consistent [4].
+- Use when there is a default value [1].
+
+### When not to use
+- Do not use in forms. Use checkboxes or radio buttons instead [2].
+- Do not use if you want to change a system state, use a toggle switch instead.
+- Do not use them to filter or navigate content, use Tabs instead [4].
```

Comment:
Est-ce que nous voulons vraiment filtrer en utilisant les Tabs?

Reply:
Dans la doc des Tabs: "_Use tabs to alternate between views within the same context, not to navigate to different areas_". Alors qu'ici tu mentionnes que nous pourrions les utiliser pour "_navigate content_".